ui/k-info-panel/edit_window.kytos

```diff
@@ -0,0 +1,718 @@
+                        }
+                        // Add the switch as an option to edit items.
+                        _this.switches_options.push({"value":switchesKeys[i], "description":description})
+                    }
+                })
+                request.fail(function(jqXHR, status) {
+                    let notification = {
+                        title: 'Error getting switches',
+                        description: 'Error getting switches for editing',
+                    }
+                    // Notify failure.
+                    _this.$kytos.$emit("setNotification" , notification);
+                })
+            },
+            /*
+                Get the interfaces from the topology API.
+            */
+            loadInterfacesForEditing: function() {
+                _this = this
+
+                // Request the interfaces from the topology API.
+                var request = $.ajax({
+                    url: this.$kytos_server_api + "kytos/topology/v3/interfaces",
+                    type: "GET",
+                    dataType: "json",
+                    contentType: "application/json"
+                })
+                request.done(function(data) {
+                    interfaces = data.interfaces
+                    interfacesKeys = Object.keys(interfaces)
+
+                    // For every interface in the topology API.
+                    for(let i = 0; i < interfacesKeys.length; i++) {
+                        description = interfacesKeys[i]
+                        try {
+                            interfaceMetadata = interfaces[interfacesKeys[i]].metadata
+                            // Check to avoid 'undefined' data.
+                            if(interfaceMetadata.hasOwnProperty('port_name')) {
+                                description = interfaces[interfacesKeys[i]].metadata.port_name
+                            }
+                        } catch(error) {
+                            // Description is id
+                        }
+                        // Add the interface as an option to edit items.
+                        _this.interfaces_options.push({"value":interfacesKeys[i], "description":description})
+                    }
+                })
+                request.fail(function(jqXHR, status) {
+                    let notification = {
+                        title: 'Error getting interfaces',
+                        description: 'Error getting interfaces for editing',
+                    }
+                    // Notify failure
+                    _this.$kytos.$emit("setNotification" , notification);
+                })
+            },
+            /*
+                Determines what can be edited and what can not be edited
+                in a maintenance window.
+            */
+            setEditable: function(data) {
+                let editable = {
+                    "Id": false,
+                    "Start": true,
```

Comment:
In the future, it would be great to have datetime pickers for both start and end values, no need to change here though, I think the creation of the maintenance could also benefit from this component if we managed to have one, cc'ing @rmotitsuki for his info too. maybe we can map this in the backlog?